onnxruntime/core/providers/dml/dml_provider_factory.cc

```diff
@@ -91,13 +110,17 @@ std::shared_ptr<IExecutionProviderFactory> CreateExecutionProviderFactory_DML(in
 }  // namespace onnxruntime
 
 ORT_API_STATUS_IMPL(OrtSessionOptionsAppendExecutionProvider_DML, _In_ OrtSessionOptions* options, int device_id) {
+API_IMPL_BEGIN
```

Comment:
[minor] Consistent indentation.
(sheesh, indentation styles in ORT seems so wonky and inconsistent)

Reply:
The ORT code is auto-formatted, so the indents wouldn't survive...

Reply:
Ugh, auto formatters are the bane of readability.

---
In reply to: [370344645](https://github.com/microsoft/onnxruntime/pull/2896#discussion_r370344645) [](ancestors = 370344645)